File: Work/Conditions/Agechecker.py
#Ryan R, Age Checker

print("Welcome to the Age checker")
age = int(input("What is your age?\n").strip())

if age < 6:
    print("You are not eligible for anything on this progam.")
elif age < 15:
    print("You can go to school")
elif age < 16:
    print("You can get a learners permit")
elif age < 18:
    print("You can drive (assuming you have a licence)")
else:
    print("You can vote in the USA!")

File: Work/Conditions/AgeChecker.c
//Ryan Rowbury, Age Checker
#include <stdio.h>
int age;

int main(void){
    printf("What is your age (Number Form)\n");
    scanf("%d", &age);
    if(age < 5){printf("You are not eligible for anything on this progam.");
    }else if (age < 15){
        printf("You can go to school");
        }else if (age < 16){
        printf("You can get a learners permit");
        }else if (age < 18){
        printf("You can drive (assuming you have a licence)");
        }else{
    printf("You can vote in the USA!");
    }
}